contentcuration/contentcuration/frontend/channelEdit/components/edit/EditModal.vue

```diff
@@ -680,4 +680,8 @@
     margin-top: -4px !important;
   }
 
+  /deep/ .v-dialog__content {
```

Comment:
@rtibbles noting here that linting doesn't seem to recorgnize ::v-deep (`Unexpected unknown pseudo-element selector`) So reverted to `/deep/` for now

Reply:
Upgrade done a few weeks ago so should be good here.